workbench/shared.php

```diff
@@ -978,15 +922,15 @@ function getComparisonOperators() {
 // color codes for status cell in the display table for job details
 function fillStatusCell($v, $jobId) {
     if(strcmp($v,'Complete')==0) {
-         echo "<td style='color:ForestGreen;font-weight:bold' id='".$jobId."_status"."'>".$v."</td></tr>";
```

Comment:
@vazexqi I've tried fixing these white space issues but the changes don't seem to make a difference or I can't seem to get the editor to change one space at a time for example. Do you have any suggestions on how to efficiently fix white space issues?